.github/actions/e2e/env-setup/action.yml

```diff
@@ -30,7 +30,7 @@ runs:
       uses: actions/cache@v4
       with:
         path: ./vendor
```

Comment:
This line hasn't been changed in this PR, but should we take the advantage to update the `path` value from this workflow to `~/.cache/composer/` to match the others? This is the only one using `vendor`, and I've read that caching the global composer cache is considered a good practice, and seems to be more efficient in terms of space.

Reply:
@alopezari On E2E tests env setup action we cache the vendor directory because we're caching the installed dependencies where as rest of the workflows cache the package files downloaded by composer but they  need run composer install. 

The [cache handling for E2E setup prevents running composer install](https://github.com/Automattic/woocommerce-payments/blob/73db2af5cfbeb676c0844d5b002c15cfb70e7a5d/.github/actions/e2e/env-setup/action.yml#L44-L47) and restores the whole vendor directory, but rest of the workflows are not configured this way. We can look at unifying this as another task. WDYT?

Reply:
Sure, sounds good!